### ISSUE TITLE
[BUG] Loading Weapons on same mount share Loaded trait

### ISSUE THREAD

**Issue by Alkyama** [Eranziel/foundryvtt-lancer#277]
**Describe the bug**
When a mount has two weapons with **Loading** tag, both of them loose the **Load** state when being used, rolling only once. Also, when clicking on the Load marker on one of the weapons, the state change is mirrored to the other weapon.

**To Reproduce**
1. Add two auxiliary weapons to a player Mech's mount with the Loading tag [e.g.: Hand Cannon].
2. Click on the roll for the weapon.
3. Check the "Loaded" state of the weapon. Both of them will be unloaded.

**Expected behavior**
Only one of the weapons should be unloaded, or it should roll twice before unloading.

**Screenshots**
Single roll, but two weapons have been loaded.
![image](https://user-images.githubusercontent.com/25304929/129121846-971faa62-6111-4f0a-8974-b5145179eaa0.png)

**Platform (please complete the following information):**
 - OS: MacOS X Big Sur 11.5
 - Browser: Google Chrome V92.0.4515.131
 - Foundry version: v0.8.8
 - Lancer system version: v0.9.6

**Additional context**
- no additional context

**Discord contact info**
alkyama#2737

**Comment by Grygon** [comment]
I though there was a commit that fixed this already but I can't find it so maybe I'm going crazy-- @whitespine thoughts?

**Comment by Eranziel** [comment]
Duplicate of #249.
